Cargo.toml

```diff
@@ -70,6 +70,9 @@ version = "0.4"
 version = "0.14"
 features = [ "http1", "runtime", "server", "tcp" ]
 
+[dependencies.indexmap]
```

Comment:
I don't see `IndexMap` actually being used, other than it being added to the TOML; am I missing something?

Reply:
Ah good catch! Removed.